docs/explanations/decisions/0018-forbid-private-member-access-in-tests.md

```diff
@@ -0,0 +1,20 @@
+# 18. Forbid Private Member Access in Tests
```

Comment:
Should this be `Forbid Private Member Access in Production Code`?

Reply:
It's a bit confusing but "Accepted" in a PR really means "Accepted on merge", just to save the effort of making another PR with a one-line change. In other words, criticism of the wording is most welcome, and you're correct!